### DISCUSSION TITLE
Question about "Floating Control"

### DISCUSSION THREAD

**Discussion by ImmersiveTheatres** [smplayer-dev/smplayer#1061] [Q&A]
First, some quick background info: I operate mobile planetariums and utilise the dual display capabilities of my MacBook (ie. laptop = primary display, and projector [dome visuals] = secondary display). 

While teaching, it's important for me to scroll through a fulldome (360°) video, without the video player controls appearing on the dome (and spoiling the immersive effect of the visuals).

The past five years I've used a MacBook Pro with a touchbar, which has allowed me to scroll through dome visuals without activating the video player controls on-screen (I've been using mpv, which produces excellent quality video for my purposes).

I've just purchased a new MacBook Pro, and the touchbar is no longer available.

So in making enquiries about a video player that will allow me to keep the video controls on the primary display (laptop), while the video plays on the secondary display (projector / dome) - someone in the mpv forum suggested SMPlayer, saying it has that option.

So I've installed the latest SMPlayer on my new MacBook Pro (macOS v15.0), and I've been experimenting (and have become very confused) with the:

- control bar
- mini control bar
- floating control

Do those three terms refer to the same thing ie. the video controls? Or are they three separate things I should be able to access?

Essentially I need to know please: **is there such a thing as an actual 'floating control bar' that I can drag from the secondary display (where the video is playing) into the primary display (laptop screen)?**

Any advice would be greatly appreciated.

**Comment by smplayer-dev** [comment]
I think you need to use the option 'Send video to screen' in the Video menu. That will display the video on one screen and the controls on another.
Alternatively, you can enable the option 'Run MPV in its own window' in Preferences → Advanced → MPlayer/MPV, which will display the video in a separate window that you can move to another screen.

**Comment by ImmersiveTheatres** [comment]
Thank you @smplayer-dev, but I've now found a simpler solution for my needs. Seeing as I love the old MacBook Pro touchbar so much, it struck me that surely there must be a software simulation of it - and there are, many! I've gone with this one and it's just perfect: https://redsweater.com/touche ✨
